modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -113,6 +113,14 @@ cat "$KUBECONFIG"
 
 .. Copy the `kubeconfig` file to all master hosts and move it to `/etc/kubernetes/kubeconfig`.
 
+.. Get the CA certificate used to validate connections from the API server.
++
+----
+# oc get configmap kube-apiserver-to-kubelet-client-ca -n openshift-kube-apiserver-operator --template='{{ index .data "ca-bundle.crt" }}' > /etc/kubernetes/ca.crt
+----
+
+.. Copy the `ca.crt` file to all other master hosts and move it to `/etc/kubernetes/ca.crt`.
```

Comment:
this should go to nodes as well

Reply:
Maybe phrase it in lines of: "Copy `/etc/kubernetes/ca.crt` to all other masters and nodes."

Reply:
Updated.